### PR TITLE
Add a UI test for caseworker good detail page

### PR DESCRIPTION
**Pull request by currycoder**
This change adds a UI test to ensure that the caseworker good detail page is working end to end.

### CODE REVIEW

File: ui_tests/caseworker/features/view_goods.feature [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@all @internal @view_goods
+Feature: I want to be able to view goods on a case
+
+  Scenario: View goods detail page
+    Given I sign in to SSO or am signed into SSO
+    And I create standard application or standard application has been previously created
+    When I go to application previously created
+    And I assign the case to "FCDO Cases to Review" queue
+    And I go to my profile page
+    And I change my team to "FCDO" and default queue to "FCDO Cases to Review"
```
Comment (saruniitr) [inline ui_tests/caseworker/features/view_goods.feature:10]: this is not really necessary but no harm in assigning to a queue

File: ui_tests/caseworker/step_defs/test_view_goods.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+@when("I click the first good on the case")  # noqa
+def click_good_on_case(driver, context):
+    driver.find_element_by_link_text(context.goods[0]["good"]["name"]).click()
+
+
+@then("I see the good details")  # noqa
+def good_details_visible(driver, context):
+    assert driver.find_element_by_tag_name("h2").text == "Product details"
+    assert driver.find_element_by_css_selector("div.govuk-caption-l").text == context.goods[0]["good"]["name"]
```
Comment (saruniitr) [inline ui_tests/caseworker/step_defs/test_view_goods.py:14]: these methods are deprecated and we are replacing them with their updated counter parts where possible.

Please see,
```
Author: Will Keeling <will@zifferent.com>
Date:   Tue Apr 12 15:49:38 2022 +0100

    Use non-deprecated methods


```

File: ui_tests/caseworker/features/view_goods.feature (hunk 1 above)
Comment (kevincarrogan) [reply ui_tests/caseworker/features/view_goods.feature:10]: I kind of think the smaller the surface area for a test the better so maybe we should just remove this.

If we skipped this we would just check on the "All cases" queue right @saruniitr?

Comment (saruniitr) [reply ui_tests/caseworker/features/view_goods.feature:10]: yes that's correct as the default team is Admin.
The existing routing rules will route it to Licensing Reception when application is submitted so we could check it from that queue as well.

**Review by saruniitr — APPROVED** [review]

**Pull request merged**